macro/Markdown.py

```diff
@@ -0,0 +1,17 @@
+import pypandoc
+import requests
+import re
+
+GITHUB_PATTERN = re.compile(r'https?://w*\.?github.com/([^/]+)/([^/]+)/blob/([^/]+)/(.*)')
+RAW_GITHUB_TEMPLATE = 'https://raw.githubusercontent.com/%s/%s/%s/%s'
```

Comment:
All this logic is kind of helpful, but I think that it will be the most fragile part of this going forward so I'd like to avoid doing the remapping and train people to put in the raw urls themselves, as they would on all the other services. gist, bitbucket, gitlab, ...